staging/src/k8s.io/apiserver/pkg/authorization/authorizer/interfaces.go

```diff
@@ -20,6 +20,7 @@ import (
 	"net/http"
 
 	"k8s.io/apiserver/pkg/authentication/user"
+	authorizationapi "k8s.io/kubernetes/pkg/apis/authorization"
```

Comment:
Don't know the implications of having this start to import an API package since before it was nicely self contained. cc @deads2k any thoughts here?